dokan-wpml.php

```diff
@@ -207,6 +218,21 @@ public function plugins_loaded() {
 
         add_action( 'wp_enqueue_scripts', [ $this, 'enqueue' ] );
 	}
+    
+    /**
+     * Clear cache when option is updated, added, or deleted
+     * This function accepts variable parameters to work with all three hooks
+     * 
+     * @param string $option_name Name of the option
+     * @param mixed  $param2      Old value (for updated_option) or value (for added_option) - optional
+     * @param mixed  $param3      New value (for updated_option) - optional
+     */
+    public function clear_option_cache( $option, $param2 = null, $param3 = null ) {
+        // Clear only if dokan_pages option is affected
+        if ( $option === 'dokan_pages' ) {
```

Comment:
Can we rethink for clear cache data. What if, we called the `get_raw_option` using a different value (except dokan_pages)?